src/frontend/css/commons.less

```diff
@@ -622,3 +622,49 @@ html:not(.ie9) .select::before, .select::before {
 .white-space-pre-line {
   white-space: pre-line;
 }
+
+/*
+ * Bouncing effect for loading animation
+ */
+.bounce {
```

Comment:
This deserves a dedicated file and the name should be more meaningful

Reply:
`animations.less`?

Reply:
`animation` is the way (technical solution) that we use to create animation loading icons. The name should represent the feature (reasonable)

For example: `loading-icon`